interpretation/deepseismic_interpretation/azureml_pipelines/README.md

```diff
@@ -0,0 +1,174 @@
+# Integrating with AzureML
+
+## Running a Pipeline in AzureML
+Set the following environment variables:
```

Comment:
Suggest you just start with .env file and explain how to populate it. I would create a "dotenv" template and then tell the user to `cp dotent .env` and then edit it

Reply:
I have included an example in this commit https://github.com/microsoft/seismic-deeplearning/pull/195/commits/3b7c5419c6332c8e082291696c2d79a7337914ef. I've also done this for .azureml/config.json in https://github.com/microsoft/seismic-deeplearning/pull/195/commits/f1285851bb34ef6e379bbc60c2cd3c261f6da029 and updated documentation in the README.md to guide users through this here https://github.com/microsoft/seismic-deeplearning/pull/195/commits/279f3367c38369ab21eb418b502b76c6849a4af3. Do you think this is an appropriate level of guidance?